test/smoke/spec/iac/snyk_test_local_exec_spec.sh

```diff
@@ -158,6 +158,23 @@ Describe "Snyk iac test --experimental command"
       The output should include "Testing pod-invalid.yaml..."
       The output should include "Failed to detect Kubernetes file, missing required fields"
     End
+
+    It "limits the depth of the directories"
+      When run snyk iac test ../fixtures/iac/depth_detection/ --experimental --detection-depth=2
+      The status should be success # no issues found
+      # Only File
+      The output should include "Testing one.tf..."
+      The output should include "Infrastructure as code issues:"
+      The output should include "Tested one.tf for known issues, found 0 issues"
```

Comment:
```suggestion
      The output should include "Tested one.tf for known issues, found"
```
We are trying to make the smoke tests less explicit and not assert against a specific number of issues:
every time the experimental command runs, the latest policies are downloaded. If in the future we add more policies which detect an issue, but we still expect 0, this regression test will fail.